packages/mbed-host-tests/mbed_host_tests/host_tests_plugins/module_copy_jlink.py

```diff
@@ -0,0 +1,23 @@
+"""
+mbed SDK
+Copyright (c) 2011-2015 ARM Limited
```

Comment:
Would you mind updating the copyright year here?
```suggestion
Copyright (c) 2019 ARM Limited
```

Reply:
Also line 17 to be removed. Plus files should have SPDX identifier (any new file should).